packages/salesforcedx-apex-replay-debugger/src/commands/apexExecutionOverlayResultCommand.ts

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { BaseCommand } from '@salesforce/salesforcedx-utils-vscode/out/src/requestService';
+import { SOBJECTS_URL } from '../constants';
+
+export interface ApexExecutionOverlayResultCommandFailure {
+  message: string;
+  errorCode: string;
+}
+
+export interface ApexExecutionOverlayResultCommandSuccess {
+  attributes: Attributes;
+  Id: string;
+  IsDeleted: boolean;
+  CreatedDate: Date;
+  CreatedById: string;
+  LastModifiedDate: Date;
+  LastModifiedById: string;
+  SystemModstamp: Date;
+  UserId: string;
+  RequestedById: string;
+  OverlayResultLength: number;
+  HeapDump: HeapDump;
+  ApexResult: HeapDumpApexResult | null;
+  SOQLResult: HeapDumpSOQLResult | null;
+  Line: number;
+  Iteration: number;
+  ExpirationDate: Date;
+  IsDumpingHeap: boolean;
+  ActionScript: string | null;
+  ActionScriptType: string;
+  ClassName: string;
+  Namespace: string;
+}
+
+export interface HeapDump {
+  className: string;
+  extents: HeadpDumpExtents[];
+  heapDumpDate: Date;
+  namespace: string;
+}
+
+export interface HeadpDumpExtents {
+  collectionType: string | null;
+  count: number;
+  definition: HeadpDumpCollectionTypeDefinition[];
+  extent: HeadpDumpExtent[];
+  totalSize: number;
+  typeName: string;
+}
+
+export interface HeadpDumpCollectionTypeDefinition {
+  name: string;
+  type: string;
+}
+
+export interface HeadpDumpExtent {
+  address: string;
+  size: number;
+  symbols: string[] | null;
+  value: HeadpDumpExtentValue;
+}
+
+// The Extent value has to be any. The value
```

Comment:
I think you're missing some stuff in this comment.

Reply:
Updated.